solarforecastarbiter/reports/templates/template.md

```diff
@@ -93,58 +94,17 @@ These intervals were removed from the raw time series before resampling and real
 
 Metrics are displayed in tables and figures below for one or more time periods. Metrics may be downloaded in csv format.
 
-### Total analysis period
-
-Metrics for the total analysis period are displayed in tables and figures below.
-
 {{ tables | safe }}
```

Comment:
This should go under the total heading

Reply:
It should already be in "metrics_total.md".  What am I missing?

Reply:
Sorry, should have been more specific. ``{{ tables | safe }}`` should go in metrics_total.md for now since the only table is a table of metrics for the total

Reply:
oh i see